app/models/sample.rb

```diff
@@ -272,12 +272,14 @@ def initiate_s3_cp
       stderr_array << stderr unless status.exitstatus.zero?
     end
     unless stderr_array.empty?
-      LogUtil.log_err_and_airbrake("Failed to upload sample #{id} with error #{stderr_array[0]}")
+      LogUtil.log_err_and_airbrake("Failed to upload S3 sample #{name} (#{id}): #{stderr_array[0]}")
```

Comment:
nit: maybe something to enclose the sample name like `'#{name}' ?

Reply:
👍 ty